scripts/asm-installer/cloudbuild-master.yaml

```diff
@@ -187,7 +187,7 @@ steps:
       --PROJECT_ID "${PROJECT_ID}"
       --BUILD_ID "${BUILD_ID}"
     env:
-    - '_LT_CLUSTER_NAME=long-term-test-cluster-environ'
+    - '_LT_CLUSTER_NAME=long-term-test-cluster-environ-newtwo'
```

Comment:
Shall we give a better name, e.g. long-term-test-cluster-environ-vm

Reply:
ok